packaging/build/Microsoft.NET.Build.Containers.targets

```diff
@@ -64,7 +64,7 @@
         <ItemGroup Label="ASP.NET port forwarding" Condition="'$(_IsAspNet)' == 'true'">
             <ContainerPort Include="80" Type="tcp" Condition="@(ContainerPort->WithMetadataValue('Identity', '80')->AnyHaveMetadataValue('Type', 'tcp')) == ''" />
 
-            <ContainerEnvironmentVariable Include="ASPNETCORE_URLS" Value="http://localhost:5000;https://localhost:5001"
+            <ContainerEnvironmentVariable Include="ASPNETCORE_URLS" Value="http://+:80"
```

Comment:
This is the biggest thing I need a sanity check on. I stole this value from the default I saw when inspecting images.

This needed to change because of the error I was seeing [in this comment](https://github.com/dotnet/sdk-container-builds/pull/181#issuecomment-1276609975)

Reply:
That's completely fine - though at this exact moment in time all of our base images already have this environment variable (transitively via the dotnet/runtime-deps base images).